src/App.js

```diff
@@ -1,12 +1,12 @@
-import React, { Component } from 'react';
-import DataTable from './DataTable';
-import './App.css';
+import React, { Component } from "react";
+import DataTable from "./DataTable";
+import "./App.css";
 
 class App extends Component {
   render() {
     return (
-      <div className="container mt-3">
-        <DataTable rows={this.props.rows} locale="da" rowsPerPage={5} />
```

Comment:
I don't see `locale` is used anywhere, unless there are some unknown reasons to me, I suggest remove it. 
I suggest to let `DataTable` maintain its own configs, it is not necessary to pass down global config IMO. App only passes down rows to `DataTable` :)